crates/trie/trie/src/proof/blinded.rs

```diff
@@ -135,6 +145,8 @@ where
     H: HashedCursorFactory + Clone + Send + Sync,
 {
     fn blinded_node(&mut self, path: &Nibbles) -> Result<Option<RevealedNode>, SparseTrieError> {
+        let start = Instant::now();
```

Comment:
we call this a bunch because this is called in a loop.

imo this clock overhead is too expensive to perform on this level

Reply:
it's called in a loop but only for those nodes that are blinded, and we usually don't have a lot of them that needs to be revealed. Nevertheless, added a check for the TRACE level for target to be enabled, so that we don't do `Instant::now` when not needed.